File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """

        pos=0

        for i in range (1,len(nums)):
            if nums[pos]!=nums[i]:
                nums[pos+1]=nums[i]
                pos+=1
            
        return pos+1

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int i =0 ;
        for ( int j = 1 ; j< nums.length;j++){
            if (nums[i]!=nums[j]){
                nums[i+1]=nums[j];
                i+=1;
            }
        }
        return i+1;

    }
}